pkg/cli/rm.go

```diff
@@ -66,6 +67,9 @@ func (a *Rm) Run(cmd *cobra.Command, args []string) error {
 			App: true,
 		}
 	}
+	if rmObjects.App && !rmObjects.Secret && !rmObjects.Volume {
+		a.Force = true //Do not prompt when deleting non-nested resource
+	}
```

Comment:
nit: just a personal preference, please ignore if you disagree.
```suggestion
	// Do not prompt when deleting non-nested resource
	a.Force = rmObjects.App && !rmObjects.Secret && !rmObjects.Volume
```